src/main/java/io/games/simple/webapp/configuration/WebSecurityConfiguration.java

```diff
@@ -20,7 +20,7 @@
     @Override
     protected void configure(HttpSecurity http) throws Exception{
         http.authorizeRequests()
-                .antMatchers("/h2-console/**", "/save").permitAll()
+                .antMatchers("/h2-console/**", "/save", "/register").permitAll()
```

Comment:
should be ok for now but we have to ask krzysiu how to block register page for logged in users